src/main/java/edu/kis/powp/jobs2d/features/FeatureManager.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+
+public class FeatureManager {
+
+    private final DrawerFeature drawerFeature;
+    private final CommandsFeature commandsFeature;
+    private final DeviceUsageFeature deviceUsageFeature;
+    private final RecordingFeature recordingFeature;
+    private final DriverFeature driverFeature;
```

Comment:
The set of features should not be fixed. Instead, they should be individually added (using e.g. addFeature or registerFeature).